Reason is currently unclear, but somehow it thinks 6.0.2 is not
higher than 5? Or is there some rule that enforces == major version?
Dropping the lower bound allows it to find assimp6.

Issue could potentially be in our own assimp6 rather than in Qt given
it does sed some bits, but needs investigation.

Either way, Qt bundles assimp-6.0.2 in Qt 6.9.2 and it should work fine
with that system version.
--- a/src/core/configure.cmake
+++ b/src/core/configure.cmake
@@ -16 +16 @@
-qt_find_package(WrapQt3DAssimp 5 PROVIDED_TARGETS WrapQt3DAssimp::WrapQt3DAssimp MODULE_NAME 3dcore QMAKE_LIB qt3d-assimp)
+qt_find_package(WrapQt3DAssimp PROVIDED_TARGETS WrapQt3DAssimp::WrapQt3DAssimp MODULE_NAME 3dcore QMAKE_LIB qt3d-assimp)
@@ -23 +23 @@
-                   PACKAGES PACKAGE WrapQt3DAssimp 5
+                   PACKAGES PACKAGE WrapQt3DAssimp
